phpBB/includes/acp/acp_users.php

```diff
@@ -44,6 +44,10 @@ function main($id, $mode)
 		$user_id	= request_var('u', 0);
 		$action		= request_var('action', '');
 
+		$referer		= request_var('referer', '');
```

Comment:
I can rename it if you prefer that, I used referer because this only effects delete action (the back link on the success message more precisely and the back link in the top right corner), and not actually redirecting via headers.